gammapy/datasets/tests/test_map.py

```diff
@@ -1246,6 +1247,18 @@ def test_map_datasets_on_off_fits_io(images, tmp_path):
     assert_allclose(dataset.mask_safe, dataset_new.mask_safe)
 
 
+@requires_data()
+def test_map_datasets_on_off_checksum(images, tmp_path):
+    dataset = get_map_dataset_onoff(images)
+    Datasets([dataset]).write(tmp_path / "test.yaml", checksum=True)
+    file = open(tmp_path / "test.yaml")
+    print(file.read())
```

Comment:
is this print needed?

Reply:
no and the line above neither. Thanks :)